lib/web_ui/lib/src/engine/canvaskit/initialization.dart

```diff
@@ -5,8 +5,28 @@
 // @dart = 2.10
 part of engine;
 
-/// EXPERIMENTAL: Enable the Skia-based rendering backend.
-const bool experimentalUseSkia =
+@JS('window.flutterWebRenderer')
+external String? get customRenderer;
```

Comment:
nit: "custom renderer" sounds like this is the renderer itself. I'd call this `requestedRendererType`.

Reply:
Also, let's leave dartdocs explaining what it's for.